libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -1156,7 +1156,7 @@ DNNReturnType ff_dnn_execute_model_tf(const DNNModel *model, DNNExecBaseParams *
     }
 
     if (extract_lltask_from_task(task, tf_model->inference_queue) != DNN_SUCCESS) {
-        av_log(ctx, AV_LOG_ERROR, "unable to extract inference from task.\n");
+        av_log(ctx, AV_LOG_ERROR, "unable to extract lltask from task.\n");
```

Comment:
for log message, we can use the full name: last level task instead of lltask

Reply:
Sure